Rakefile

```diff
@@ -48,8 +167,69 @@ namespace :test do
     end
     Dir.chdir( original_dir )
   end
+
+  desc <<-EOS
+Run the base beaker acceptance tests
+#{USAGE}
+  EOS
+  task :base  => 'gen_hosts' do
+    beaker_test({:tests => 'acceptance/tests/base'})
+  end
+
+  desc <<-EOS
+Run the hypervisor beaker acceptance tests
+#{USAGE}
+  EOS
+  task :hypervisor  => 'gen_hosts' do
+    beaker_test({:tests => 'acceptance/tests/hypervisor'})
+  end
+
+  desc <<-EOS
+Run the puppet beaker acceptance tests on a pe install.
+#{USAGE}
+  EOS
+  task :puppetpe  => 'gen_hosts' do
+    beaker_test({:pre_suite => 'acceptance/pre_suite/pe/install.rb', :tests => 'acceptance/tests/puppet'})
```

Comment:
specifying the pre_suite here *might* break preserved_hosts_mode, but maybe that's ok